models/Daily_naive_darts/src/main.py

```diff
@@ -0,0 +1,74 @@
+import os
+from enfobench.dataset import Dataset
+import pandas as pd
+import numpy as np
+from darts import TimeSeries
+from darts.models import NaiveSeasonal
+from enfobench.evaluation import evaluate_metrics
+from enfobench.evaluation.metrics import mean_absolute_error, mean_bias_error,root_mean_squared_error
+from datetime import timedelta
+from enfobench import AuthorInfo, ModelInfo, ForecasterType
+from enfobench.evaluation.server import server_factory
+from enfobench.evaluation.utils import create_forecast_index
+
+class NaiveModel:
+
+    def info(self) -> ModelInfo:
+        
+        return ModelInfo(
+            name="Darts.dailyNaive",

```

Comment:
Please use CamelCase for the name to be consistent with all other models. "Darts.DailyNaive"

Reply:
I have updated the name of the model to "DailyNaiveModel"